base/setup/usetup/lang/hr-HR.h

```diff
@@ -0,0 +1,2338 @@
+// This file is converted by code7bit.
+// code7bit: https://github.com/katahiromz/code7bit
+// To revert conversion, please execute "code7bit -r <file>".
+#pragma once
+
+static MUI_ENTRY hrHRSetupInitPageEntries[] =
+{
+    {
+        4,
+        3,
+        " ReactOS " KERNEL_VERSION_STR " instalacija ",
+        TEXT_STYLE_UNDERLINE,
+        TEXT_ID_STATIC
+    },
+    {
+        0,
+        20,
+        "Pri\304\215ekajte da se instalacija u\304\215ita",
+        TEXT_STYLE_NORMAL | TEXT_ALIGN_CENTER,
+        TEXT_ID_STATIC
+    },
+    {
+        0,
+        21,
+        "i otkrije va\305\241e ure\304\221aje",
+        TEXT_STYLE_NORMAL | TEXT_ALIGN_CENTER,
+        TEXT_ID_STATIC
+    },
+    {
+        0,
+        0,
+        "Molimo pri\304\215ekajte",
+        TEXT_TYPE_STATUS | TEXT_PADDING_BIG,
+        TEXT_ID_STATIC
+    },
+    {
+        0,
+        0,
+        NULL,
+        0
+    }
+};
+
+static MUI_ENTRY hrHRLanguagePageEntries[] =
+{
+    {
+        4,
+        3,
+        " ReactOS " KERNEL_VERSION_STR " instalacija ",
+        TEXT_STYLE_UNDERLINE,
+        TEXT_ID_STATIC
+    },
+    {
+        6,
+        8,
+        "Odabir jezika",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        8,
+        10,
+        "\x07  Molimo odaberite jezik koji \304\207e se koristiti za vrijeme instalacije",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        8,
+        11,
+        "   I pritisnite ENTER",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        8,
+        13,
+        "\x07  Ovaj jezik \304\207e biti zadan za cijeli sustav",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        0,
+        0,
+        "ENTER = Nastavi  F3 = Iza\304\221i",
+        TEXT_TYPE_STATUS | TEXT_PADDING_BIG,
+        TEXT_ID_STATIC
+    },
+    {
+        0,
+        0,
+        NULL,
+        0
+    }
+};
+
+static MUI_ENTRY hrHRWelcomePageEntries[] =
+{
+    {
+        4,
+        3,
+        " ReactOS " KERNEL_VERSION_STR " instalacija ",
+        TEXT_STYLE_UNDERLINE,
+        TEXT_ID_STATIC
+    },
+    {
+        6,
+        8,
+        "Dobrodo\305\241li u ReactOS instalaciju",
+        TEXT_STYLE_HIGHLIGHT,
+        TEXT_ID_STATIC
+    },
+    {
+        6,
+        11,
+        "Ovaj dio instalacije kopira ReactOS na va\305\241e",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        6,
+        12,
+        "ra\304\215unalo i priprema se za drugi dio instalacije.",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        8,
+        15,
+        "\x07  Pritisnite ENTER za instalaciju ili nadogradnju ReactOS-a.",
+        TEXT_STYLE_NORMAL,
+        TEXT_ID_STATIC
+    },
+    {
+        8,
+        17,
+     // "\x07  Pritisnite R za popravak ReactOS instalacije koristeći Recovery Console.",

```

Comment:
The word `koristeći` contains an Unicode character. Despite it's just a comment, I'd still recommend converting it with code7bit, as it could replace existing text at some time.

Reply:
I will convert again, maybe code7bit don't convert all

Reply:
I convert again and no change:
![image](https://user-images.githubusercontent.com/83548580/139857782-6c1c1fe3-da96-40ce-b688-f8e741a87b58.png)

![image](https://user-images.githubusercontent.com/83548580/139857850-b6d09a0f-f92e-4cc4-b2c7-6c8b8f82beb3.png)

Reply:
From other usages of the letter "ć" that have been successfully converted (e.g. "će biti" etc.), I think the corresponding encoding would be: `\304\207` .

Reply:
I will fix that

Reply:
Fixed